content/docs/glossary/sailudon.md

```diff
@@ -8,6 +8,7 @@ category = "arvo"
 
 **Sail** and **Udon** are domain specific languages for Hoon.
 
-Sail is used to express XML data structures, which are used for rendering webpages, such as when browsing [collections](/docs/glossary/collection) in [Landscape](/docs/glossary/landscape).
+Sail is used to express XML data structures, which are used for rendering
+webpages. See the [Sail Guide](/docs/hoon/guides/sail) for more details.
 
 Udon is similar to Markdown, a simple language for writing documents. Udon is stored in [Clay](/docs/glossary/clay) and rendered using [Ford](/docs/glossary/ford).
```

Comment:
"Udon documents are stored in Clay and built using the `++ford` arm."  If this sentence is even necessary, which I rather think it is not.

Reply:
@tinnus-napbus do you want to put `;>` in here as well?